app/src/main/java/com/infomaniak/mail/views/InformationBlockView.kt

```diff
@@ -52,7 +53,9 @@ class InformationBlockView @JvmOverloads constructor(
 
     init {
         attrs?.getAttributes(context, R.styleable.InformationBlockView) {
-            text = getString(R.styleable.InformationBlockView_text)
+            binding.informationButton.apply {
+                setOnClickListener { onActionClicked?.invoke() }
+            }
```

Comment:
You need to support passing the missing defined attributes like title and description